config/locales/en/document_images/index.yml

```
@@ -2,7 +2,7 @@ en:
   document_images:
     index:
       title: "Images for ‘%{title}’"
-      description_govspeak: Images can be jpg, png or gif files. [Full guidance on images](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/images-and-videos)
+      description_govspeak: Images can be jpg, png or gif files. [Full guidance on images](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/images-and-videos).
```

Might be that you want to edit this to match https://github.com/alphagov/content-publisher/pull/544/files#diff-2f9eaf5b2570beef0d9be476e9a659d4R5 so @paola-roccuzzo can avoid a merge conflict

```suggestion
      description_govspeak: Images can be jpg, png or gif files. Full [guidance on images](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/images-and-videos).
```